{"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/54/comments/33693456.json"}, "html": {"href": "#!/osrf/srcsim/issues/54#comment-33693456"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/54.json"}}, "type": "issue", "id": 54, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Java Crash of 'cap' Running armDemo.py"}, "content": {"raw": "@rmerriam I am unable to reproduce the exception using the values you specified with the latest SRCSim release and armDemo.py. Can you test to see if you are still seeing this problem?", "markup": "markdown", "html": "<p>@rmerriam I am unable to reproduce the exception using the values you specified with the latest SRCSim release and armDemo.py. Can you test to see if you are still seeing this problem?</p>", "type": "rendered"}, "created_on": "2017-01-16T20:46:12.270358+00:00", "user": {"display_name": "Nathan Mertins", "uuid": "{fdf3b5ab-2811-4209-9517-7b6481ff5541}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bfdf3b5ab-2811-4209-9517-7b6481ff5541%7D"}, "html": {"href": "https://bitbucket.org/%7Bfdf3b5ab-2811-4209-9517-7b6481ff5541%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/ba4d38bd80931e6ee8db341181653cd6d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsNM-1.png"}}, "nickname": "nmertins", "type": "user", "account_id": "557058:53dfbd27-8588-4c5e-a1e4-ef7656d0ad23"}, "updated_on": null, "type": "issue_comment", "id": 33693456}